shared-libs/cht-datasource/src/person.ts

```diff
@@ -32,14 +32,23 @@ export namespace v1 {
     }
   };
 
-  const assertTypeQualifier: (
+  const assertTypeQualifier: (qualifier: unknown) => asserts qualifier is ContactTypeQualifier = (
     qualifier: unknown
-  ) => asserts qualifier is ContactTypeQualifier = (qualifier: unknown) => {
+  ) => {
     if (!isContactTypeQualifier(qualifier)) {
       throw new Error(`Invalid type [${JSON.stringify(qualifier)}].`);
     }
   };
 
+  const assertLimitAndSkip = (limit: unknown, skip: unknown) => {
+    if (typeof limit !== 'number' || limit <= 0) {
```

Comment:
Can you also include a `Number.isInteger` check? Just don't want to let anyone be passing decimal numbers here.